### PR TITLE
feat(foundations/layout): clean documentation

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

File: src/docs/Foundations/Layout/Grid/code.mdx [hunk 1]
```diff
@@ -24,43 +19,24 @@ order: 1
 @import 'layouts/_l.flexy';
 ```
 
-`ml-flexy` is very close to the bootstrap flex grid system.
+### How it works
 
-To create a grid, you need two elements:
+The below example creates four equal-width columns at small, medium, large, and extra large breakpoints using our predefined grid classes. The columns are centered in the page within the parent container.
```
Comment (tiloyi) [inline src/docs/Foundations/Layout/Grid/code.mdx:24]: ~~The columns are centered in the page within the parent container.~~ not necessary

File: src/docs/Foundations/Layout/Grid/code.mdx [hunk 2]
```diff
@@ -24,43 +19,24 @@ order: 1
...
-- Columns: `ml-flexy__col`
+<Preview path="basic" />
 
-```html
-<div class="ml-flexy">
-  <div class="ml-flexy__col">
-    col 1
-  </div>
+#### Container
+`ml-container` defines the overall grid context at the top level and applies the proper margin and our default max-width.  You can also remove this max width, by using the grid modifier class ml--container--fluid.
```
Comment (tiloyi) [inline src/docs/Foundations/Layout/Grid/code.mdx:29]: The `ml-container` plays no role in the construction of the grid. 
Therefore, I think we should put this part in a "Highlight".

On the other hand, you could replace this part with an explanation of the `ml-flexy` class.

File: src/docs/Foundations/Layout/Grid/code.mdx [hunk 3]
```diff
@@ -24,43 +19,24 @@ order: 1
...
 
-```html
-<main class="ml-container">
-  <div class="ml-flexy">
-    ...
-  </div>
-</main>
-```
+#### Columns
+`ml-flexy__col` defines individual columns inside a row. Use as many columns as you’d like out of the possible 12 per row. With Flexbox columns widths are set in percentages relative to their parent element so columns without a spec’d width will automatically be equal. For example, four instances of `ml-flexy__col` will will give you four equal width columns — each 25% of the parent from the small breakpoint on up.
```
Comment (tiloyi) [inline src/docs/Foundations/Layout/Grid/code.mdx:33]: ~~inside a row~~ => inside the grid. _(<= we don't have a "row system" inside Mozaic's Grid)_.

~~Use as many columns as you’d like out of the possible 12 per row.~~ => I think that it's not necessary.

File: src/docs/Foundations/Layout/Grid/code.mdx [hunk 4]
```diff
@@ -117,12 +81,10 @@ You can apply specific columns width using `.ml-flexy__col` modifiers:
 | `.ml-flexy__col--fill`    | fill the available width    |
 | `.ml-flexy__col--full`    | 100%                        |
 
-<br />
-<Preview path="flexy" />
 
-### Responsive widths
+### Responsive
```
Comment (tiloyi) [inline src/docs/Foundations/Layout/Grid/code.mdx:85]: ### Responsive modifiers

File: src/docs/Foundations/Layout/Grid/index.mdx [hunk 5]
```diff
@@ -7,11 +7,11 @@ status:
   scss: 'stable'
 ---
 
-> A grid is used to create a **more structured and more harmonious design.** It’s the frame of your page, you build the layout by placing blocks of contents (text, images,…) on a certain number of columns. Gutters are used to separate 2 blocks, they give air to the design. Columns and gutters form a container. The container can be **full width or with a fixed size.**
+> Mozaic Flex grid system is a fully responsive, customizable, 12 column layout system with five default breakpoints, Sass variables and mixins, and dozens of predefined classes.
```
Comment (tiloyi) [inline src/docs/Foundations/Layout/Grid/index.mdx:10]: In my opinion, we should keep the old definition, which is a good detail of how a grid works. We should just rephrase perhaps some of the sentences.

**Review by nherwegh — CHANGES_REQUESTED** [review]

File: src/docs/Foundations/Layout/Grid/code.mdx [hunk 6]
```diff
@@ -24,43 +18,25 @@ order: 1
...
 
-```html
-<main class="ml-container">
-  <div class="ml-flexy">
-    ...
-  </div>
-</main>
-```
+#### Columns
+`ml-flexy__col` defines individual columns inside the grid. With Flexbox columns widths are set in percentages relative to their parent element so columns without a spec’d width will automatically be equal. For example, four instances of `ml-flexy__col` will will give you four equal width columns — each 25% of the parent from the small breakpoint on up.
```
Comment (nherwegh) [inline src/docs/Foundations/Layout/Grid/code.mdx:33]: there are the word "will" two times

**Review by nherwegh — CHANGES_REQUESTED** [review]

**Review by nherwegh — APPROVED** [review]

File: src/docs/Foundations/Layout/Grid/index.mdx [hunk 7]
```diff
@@ -7,11 +7,11 @@ status:
   scss: 'stable'
 ---
 
-> A grid is used to create a **more structured and more harmonious design.** It’s the frame of your page, you build the layout by placing blocks of contents (text, images,…) on a certain number of columns. Gutters are used to separate 2 blocks, they give air to the design. Columns and gutters form a container. The container can be **full width or with a fixed size.**
+> A grid is used to create a structured and harmonious design for every breakpoints. It is the frame of your page that allows you to build your layout by placing blocks of contents (texts, images, etc.) using a certain number of columns. Gutter are used to give some blank space between 2 separate blocks. A container is the combination of columns and gutters. The container can be full width (fluid) or have a fixed size (fixed container).
```
Comment (caqueste) [inline src/docs/Foundations/Layout/Grid/index.mdx:10]: `(texts, images, etc.)` → Remove the dot 🙂

Comment (caqueste) [reply src/docs/Foundations/Layout/Grid/index.mdx:10]: From my point of view, this part seems useless: `Gutter are used to give some blank space between 2 separate blocks. A container is the combination of columns and gutters. The container can be full width (fluid) or have a fixed size (fixed container).`.
The grid anatomy explains the same things.

**Review by caqueste — CHANGES_REQUESTED** [review]

File: src/docs/Foundations/Layout/Grid/code.mdx [hunk 8]
```diff
@@ -24,43 +18,25 @@ order: 1
...
+</Highlight>
 
-```html
-<main class="ml-container">
-  <div class="ml-flexy">
-    ...
-  </div>
-</main>
-```
+#### Columns
```
[the remark this answers is not in the document]
Comment (mohamedMok) [reply src/docs/Foundations/Layout/Grid/code.mdx:32]: lol fair enough! 'specific'

File: src/docs/Foundations/Layout/Grid/index.mdx (hunk 7 above)
Comment (mohamedMok) [reply src/docs/Foundations/Layout/Grid/index.mdx:10]: etc. is an abbreviation that needs the dot.

**Review by caqueste — APPROVED** [review]

**Pull request merged**